### PR TITLE
Feature/profile preferred locations removal

### PR DESCRIPTION
**Pull request by yanisurbis**
#5

### CODE REVIEW

File: client/src/wh/components/forms/views.cljs [hunk 1]
```diff
@@ -152,24 +152,28 @@
...
-    (fn [value {:keys [suggestions dirty? error force-error? read-only on-select-suggestion hide-icon?] :as options}]
+    (fn [value {:keys [suggestions dirty? error force-error? read-only on-select-suggestion on-remove hide-icon?] :as options}]
       (when (and (not (nil? dirty?))
                  (boolean? dirty?))
         (reset! dirty dirty?))
       (let [value (or value (:value options))
             suggestable? (and (or (seq suggestions) on-select-suggestion) (not hide-icon?))
-            show-suggestion? (and (seq suggestions) @focused (not read-only))]
+            show-suggestion? (and (seq suggestions) @focused (not read-only))
+            removable? (and value (not @focused))]
```
Comment (acron0) [inline client/src/wh/components/forms/views.cljs]: `removable?` should also only be true if `on-remove` is a valid seq. At the moment this is causing the remove icons to appear on other text fields:
Screenshot_20190930_105029

**Comment by acron0** [comment]
https://github.com/WorksHub/client/issues/5

File: client/src/wh/components/forms/views.cljs (hunk 1 above)
Comment (yanisurbis) [reply client/src/wh/components/forms/views.cljs]: Thank you for the review and sorry for missing this

**Comment by acron0** [comment]
OK TO MERGE

**Comment by WorksHubCodi** [comment]
Thanks for contributing! This code has been merged upstream!

**Comment by acron0** [comment]
@yanisurbis Please send me an email antony@functionalworks.com :) Thanks for your help

**Pull request closed without merging**